mgmtsystem_action/models/mgmtsystem_action.py

```diff
@@ -136,15 +136,25 @@ def create(self, vals):
         """Creation of Action."""
         Sequence = self.env['ir.sequence']
         vals['reference'] = Sequence.next_by_code('mgmtsystem.action')
-        action = super(MgmtSystemAction, self).create(vals)
-        self.send_mail_for_action(action)
```

Comment:
Correction: This was not an mistake, this is removed because of the new feature introduced in project module in V10.0.

Its this method: _track_template(self, tracking):
checks if stage_id has been changed and sends the mail template that is associated to the stage.

Big advantage in comparison to the old method is that you can choose which stages send notifications for actions. In the old method the notifications were allways send regardles if you wanted it or not.